constant_sorrow/tests.py

```diff
@@ -1,10 +1,10 @@
 import pytest
+
 from constant_sorrow import constants
 
 
 def test_establishing_a_constant():
     # You get a constant by just picking an all-caps name and importing it.
-    from constant_sorrow.constants import THIS_IS_A_VALID_CONSTANT
```

Comment:
Why removing this line? It's the single most important line on this file.

Reply:
Riiight.  That was a mistake.